FireboltDotNetSdk.Tests/Integration/ExecuteTest.cs

```diff
@@ -84,6 +84,37 @@ private void executeTest(string connString, string commandText, string? addition
             }
         }
 
+        [Test]
+        public void NoTimeout()
+        {
+            using var conn = new FireboltConnection(SYSTEM_CONNECTION_STRING);
+            conn.Open();
+            DbCommand command = conn.CreateCommand();
+            command.CommandText = "select * from information_schema.tables";
```

Comment:
This query makes tests rely on I_S.tables taking a set amount of time. I feel this makes tests brittle wrt changes on the backend and/or test setup (if we have many tables in db they will be slower). I would suggest using something like `"SELECT checksum(*) FROM GENERATE_SERIES(1, 300000000000)"`, where `300000000000` governs time it executes for. Obviously a low value here would suffice.
We use this query throughout to test execution time so we know what to watch out for.